### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by noahdurbin**
- refactored update method in admin application
- refactored model tests

### CODE REVIEW

File: app/models/application.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Application < ApplicationRecord
   validates :address, presence: true
   validates :city, presence: true
   validates :state, presence: true
-  validates :zip, presence: true
+  validates :zip, presence: true, numericality: true
```
Comment (JCL461437) [inline app/models/application.rb:6]: Glad to see numerically validations here

File: app/models/pet_application.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class PetApplication < ApplicationRecord
   belongs_to :application
   belongs_to :pet
+  validates :pet_id, presence: true
+  validates :application_id, presence: true
```
Comment (JCL461437) [inline app/models/pet_application.rb:5]: Great validation fixes.

File: spec/models/pet_application_spec.rb [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe PetApplication, type: :model do
   describe 'relationships' do
-    it { should have_many(:applications) }
-    it { should have_many(:pets) }
+    it {should belong_to(:application)}
```
Comment (JCL461437) [inline spec/models/pet_application_spec.rb:5]: Great tests for validations Noah. Great work

File: spec/models/application_spec.rb [hunk 4]
```diff
@@ -3,16 +3,16 @@
...
   describe 'relationships' do
     it { should have_many(:pet_applications) }
-    it { should have_many(:pets, through: :pet_applications) }
+    it { should have_many(:pets).through(:pet_applications)}
   end
 
   describe 'validations' do
     it { should validate_presence_of(:name) }
-    it { should validate_presence_of(:adress) }
+    it { should validate_presence_of(:address) }
```
Comment (JCL461437) [inline spec/models/application_spec.rb:11]: Great refactors here

**Review by JCL461437 — APPROVED** [review]
Great work Noah, appreciate all your effort!

**Review by AuroraTuring — APPROVED** [review]

**Pull request merged**